server/graphql/catalogue/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "graphql_asset_catalogue"
```

Comment:
perhaps the folder should be `asset_catalogue`?

Reply:
I agree though :)